cmake/sdp.cmake

```diff
@@ -100,7 +104,7 @@ function(sdp_assembly_prepare_install hrt_srcs)
   string(REPLACE ";" "$<SEMICOLON>" hrt_srcs_semi "${hrt_srcs}")
 
   add_custom_target(asm_install
-    COMMAND ${CMAKE_COMMAND} -D hrt_srcs="${hrt_srcs_semi}" -P ${ZEPHYR_NRF_MODULE_DIR}/cmake/sdp_asm_install.cmake
+    COMMAND ${CMAKE_COMMAND} -D hrt_srcs="${hrt_srcs_semi}" -D target_soc="${target_soc}" -P ${ZEPHYR_NRF_MODULE_DIR}/cmake/sdp_asm_install.cmake
```

Comment:
`target_soc` is undefined if this function is called directly.

Reply:
Added as a parameter, is that okay or further validation is necessary?